.github/workflows/go_test_integration.yml

```diff
@@ -54,6 +54,7 @@ jobs:
         run: |
           export KO_DOCKER_REPO="kind.local"
           bash hack/quick-install.sh
+          bash integrations/logging/splunk/install.sh
```

Comment:
I agree with this change, makes sense that splunk is already installed before running the tests but we miss a thing to be tested in this change, that the logging configuration changes are correctly loaded after restarting the sink/api.

I'd add two more tests to cover this (they don't need to be added in this PR, it's ok to do them in a different one):
- Modify the logging-credentials, restart the api, check that the request doesn't have permissions and set it correctly again (similar to the test where we fence the database)
- Modify the logging-configuration, restart the sink, see how the URL has been changed and doesn't retrieve what's expected and reconfigure it.